storm-core/src/clj/org/apache/storm/ui/core.clj

```diff
@@ -18,7 +18,8 @@
   (:use compojure.core)
   (:use [clojure.java.shell :only [sh]])
   (:use ring.middleware.reload
-        ring.middleware.multipart-params)
+        ring.middleware.multipart-params
+        ring.middleware.multipart-params.temp-file)
```

Comment:
Nevermind. Shading makes this import necessary. Multipart-params tries to load it at runtime and blows up if it's not already in the namespace.